gradle-plugin/plugin/build.gradle.kts

```diff
@@ -108,9 +108,6 @@ val packagePerformanceProjectTemplateTask =
   }
 
 tasks["processResources"].dependsOn(packagePerformanceProjectTemplateTask)
-afterEvaluate {
```

Comment:
I removed this since the publishPluginJar task doesn’t exist.
If `processResources` already depends on `packagePerformanceProjectTemplateTask` this seems redundant.